lib/fizzy/instantiate.hpp

```diff
@@ -87,13 +87,22 @@ struct Instance
 };
 
 // Instantiate a module.
-std::unique_ptr<Instance> instantiate(Module module,
+// Transfers ownership of passed module to instance.
+std::unique_ptr<Instance> instantiate(std::unique_ptr<Module>&& module,
```

Comment:
Another possible option would be to have 
- for moving:  `std::unique_ptr<Instance> instantiate(std::unique_ptr<Module> module,`
- for copying: `std::unique_ptr<Instance> instantiate(const Module& module,`

but that would require even more changes to the tests to do either `instantiate(std::move(module), ...` or `instantiate(*module, ...`

I would better do it in a follower PR if it's a preferable variant.

Reply:
If the copy is to be made, better API is `instantiate(Module module, ...)`